crates/ruff_linter/resources/test/fixtures/pyflakes/F401_24/__init__.py

```diff
@@ -0,0 +1,47 @@
+'''__init__.py without __all__
+
+Unused stdlib and third party imports are unsafe removals
+
+Unused first party imports get changed to redundant aliases
+'''
+
+
+
+# stdlib
+
+
+
+import os # Ok: is used
+_ = os
+
+
+
+import argparse as argparse # Ok: is redundant alias
+
+
+
+import sys # F401: remove unused
+
+
+
+# first-party
+
+
+
+from . import used # Ok: is used
+_ = used
+
+
+
+from . import aliased as aliased # Ok: is redundant alias
+
```

Comment:
What's up with the crazy spacing in this file?

Reply:
Presuming this is to make the snapshots clearer, but I don't think we do this elsewhere I would stick with the existing style

Reply:
Yeah, I was having trouble reading the snapshots. Willfix.